modules/@apostrophecms/ui/ui/apos/components/AposButton.stories.js

```diff
@@ -66,15 +69,18 @@ export const buttons = () => ({
         )
     }
   },
+  // Margin added to make room for demonstrating tooltip.
   template: `
     <AposButton
+      style="margin: 100px;"
       :disabled="disabled"
       :label="label"
       :busy="busy"
       :type="type"
       :icon="icon"
       :iconOnly="iconOnly"
       v-bind:modifiers="modifiers"
+      :tooltip="tooltip ? 'Duis mollis, est<br />porttitor ligula.' : null"
```

Comment:
Looks like we need to use a `br` tag for line breaks.

Reply:
It looks like we could alternatively set a max-width on the inner bit and v-tooltip will figure out positioning.